home/vlt-adm/system/network-ips.sh

```diff
@@ -6,11 +6,15 @@ if [ "$(/usr/bin/id -u)" != "0" ]; then
    exit 1
 fi
 
-ip="$1"
+management_ip="$1"
+internet_ip="$2"
+backends_outgoing_ip="$3"
+logom_outgoing_ip="$4"
 
-/sbin/ifconfig | grep "$ip" > /dev/null
+/sbin/ifconfig | grep "$management_ip" > /dev/null
 if [ "$?" == 0 ]; then
-    /bin/echo "$ip" > /usr/local/etc/management.ip
+    # update node network ips in /etc/rc.conf.d/network
+    /usr/sbin/sysrc -f /etc/rc.conf.d/network management_ip=$management_ip internet_ip=$internet_ip backends_outgoing_ip=$backends_outgoing_ip logom_outgoing_ip=$logom_outgoing_ip
```

Comment:
Condition is not in sync with what needs to be modified: the user might only modify `backends_outgoing_ip` and parameters in the system won't be changed
This needs to be adapted (and separate configuration files be regenerated, especially pf.conf)